### DISCUSSION TITLE
Possibility to override determined type

### DISCUSSION THREAD

**Discussion by cur3n4** [mrjono1/joi-to-typescript#479] [General]
Hello, great library! 

One question, currently `Joi.date()` is generated as `Date`. Is there a possibility to override the generated type to something else (i.e. `string`)?

Another feature that would be nice is to be able to specify a prefix for the interface file name `interfaceFilePrefix`, in the same way it works for suffix.

Thanks!

**Comment by mrjono1** [comment]
this should do it

```
export const TestSchema = Joi.object({
  data: Joi.date().required().meta({ className: 'string' })
}).meta({ className: 'Test' });
```

**Reply by cur3n4** [reply]
Perfect, thanks a lot!
